### PR TITLE
Six new flaky tests in : https://github.com/apache/hive ,serde,ecdac363e042200d85bb8b894aa20d91a1642a18

### PR DESCRIPTION
**Pull request by KobeNorris**
## Details
* **Six New Failure Tests** with NonDex 1.1.2 on SHA ecdac363e042200d85bb8b894aa20d91a1642a18 【Latest】
* nondexMode=FULL nondexSeed=933178
* nondexMode=FULL nondexSeed=974622
* nondexMode=FULL nondexSeed=1016066

## Specific Tests with test blocks
```
https://github.com/apache/hive ; serde ; ecdac363e042200d85bb8b894aa20d91a1642a18 ; org.apache.hadoop.hive.serde2.TestStatsSerde.testLazyBinarySerDe
https://github.com/apache/hive ; serde ; ecdac363e042200d85bb8b894aa20d91a1642a18 ; org.apache.hadoop.hive.serde2.binarysortable.TestBinarySortableSerDe.testBinarySortableSerDe
https://github.com/apache/hive ; serde ; ecdac363e042200d85bb8b894aa20d91a1642a18 ; org.apache.hadoop.hive.serde2.columnar.TestLazyBinaryColumnarSerDe.testSerDeEmpties
https://github.com/apache/hive ; serde ; ecdac363e042200d85bb8b894aa20d91a1642a18 ; org.apache.hadoop.hive.serde2.columnar.TestLazyBinaryColumnarSerDe.testSerDe
https://github.com/apache/hive ; serde ; ecdac363e042200d85bb8b894aa20d91a1642a18 ; org.apache.hadoop.hive.serde2.columnar.TestLazyBinaryColumnarSerDe.testLazyBinaryColumnarSerDeWithEmptyBinary
https://github.com/apache/hive ; serde ; ecdac363e042200d85bb8b894aa20d91a1642a18 ; org.apache.hadoop.hive.serde2.columnar.TestLazyBinaryColumnarSerDe.testSerDeInnerNulls
```

### CODE REVIEW

File: pr-data.csv [hunk 1]
```diff
@@ -1283,17 +1283,23 @@ https://github.com/apache/hive,90fa9064f2c6907fbe6237cb46d5937eebd8ea31,standalo
 https://github.com/apache/hive,90fa9064f2c6907fbe6237cb46d5937eebd8ea31,standalone-metastore/metastore-server,org.apache.hadoop.hive.metastore.TestPartitionManagement.testPartitionRetention,ID,,,
 https://github.com/apache/hive,90fa9064f2c6907fbe6237cb46d5937eebd8ea31,standalone-metastore/metastore-server,org.apache.hadoop.hive.metastore.utils.TestMetaStoreServerUtils.testGetPartitionspecsGroupedBySDonePartitionCombined,ID,Rejected,https://github.com/apache/hive/pull/2719,
 https://github.com/apache/hive,54e43339dd671018fc70ebb5d9f0b292d70391a6,itests/serde,org.apache.hadoop.hive.serde2.avro.TestAvroDeserializer.canDeserializeTimestamps,OD,,,
-https://github.com/apache/hive,90fa9064f2c6907fbe6237cb46d5937eebd8ea31,serde,org.apache.hadoop.hive.serde2.columnar.TestLazyBinaryColumnarSerDe.testHandlingAlteredSchemas,ID,Rejected,https://github.com/apache/hive/pull/923,
```
Comment (darko-marinov) [inline pr-data.csv:1286]: Do not delete these cases where the status is not empty, e.g., `Rejected` or `Accepted`.

File: pr-data.csv [hunk 2]
```diff
@@ -1283,17 +1283,23 @@ https://github.com/apache/hive,90fa9064f2c6907fbe6237cb46d5937eebd8ea31,standalo
...
-https://github.com/apache/hive,90fa9064f2c6907fbe6237cb46d5937eebd8ea31,serde,org.apache.hadoop.hive.serde2.lazy.TestLazySimpleSerDe.testSerDeParameters,ID,,,
```
Comment (darko-marinov) [inline pr-data.csv:1287]: Do not change just the commit SHA. It's good that you confirmed that the test is still flaky, but let's keep the original SHA where it was found. You can add new tests that are not already in the file.

File: pr-data.csv [hunk 3]
```diff
@@ -1283,17 +1283,23 @@ https://github.com/apache/hive,90fa9064f2c6907fbe6237cb46d5937eebd8ea31,standalo
...
+https://github.com/apache/hive,ecdac363e042200d85bb8b894aa20d91a1642a18,serde,org.apache.hadoop.hive.serde2.objectinspector.TestProtocolBuffersObjectInspectors.testProtocolBuffersObjectInspectors,ID,,,
+https://github.com/apache/hive,ecdac363e042200d85bb8b894aa20d91a1642a18,serde,org.apache.hadoop.hive.serde2.objectinspector.TestReflectionObjectInspectors.testReflectionObjectInspectors,ID,,,
+https://github.com/apache/hive,ecdac363e042200d85bb8b894aa20d91a1642a18,serde,org.apache.hadoop.hive.serde2.objectinspector.TestStandardObjectInspectors.testStandardUnionObjectInspector,ID,Accepted,https://github.com/apache/hive/pull/929,
+https://github.com/apache/hive,ecdac363e042200d85bb8b894aa20d91a1642a18,serde,org.apache.hadoop.hive.serde2.objectinspector.TestThriftObjectInspectors.testThriftObjectInspectors,ID,,,
+https://github.com/apache/hive,ecdac363e042200d85bb8b894aa20d91a1642a18,serde,org.apache.hadoop.hive.serde2.objectinspector.TestThriftObjectInspectors.testThriftSetObjectInspector,ID,,,
+https://github.com/apache/hive,ecdac363e042200d85bb8b894aa20d91a1642a18,serde,org.apache.hadoop.hive.serde2.TestStatsSerde.testLazyBinarySerDe,ID,,,
```
Comment (darko-marinov) [inline pr-data.csv:1302]: Again, don't delete the info that's already there, but it's fine to add a new test.

**Review by darko-marinov — CHANGES_REQUESTED** [review]
It's great that you found some new tests, and it's good that you confirmed some existing tests. Please do not change the existing tests just to update the commit SHA. That easiest is to just add new tests that are not already in the file.

File: pr-data.csv (hunk 1 above)
Comment (KobeNorris) [reply pr-data.csv:1286]: Sorry about that, I think I misunderstood the third requirement for contribution

File: pr-data.csv (hunk 3 above)
Comment (KobeNorris) [reply pr-data.csv:1302]: I will restore the file and update them again

**Review by darko-marinov — CHANGES_REQUESTED** [review]
Thanks for the updates. The file change looks fine now. Can you squash your 8 commits into 1? If you don't know how to do it, I can do it.

**Comment by KobeNorris** [comment]
I will take a try

**Comment by KobeNorris** [comment]
I did it by force push, so it is a little bit dirty..

**Comment by darko-marinov** [comment]
It'e perfectly fine; force push was necessary. Accepting...

**Comment by KobeNorris** [comment]
Thanks!

**Pull request merged**